core/Updates/5.0.0-b1.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Updates;
+
+use Piwik\DbHelper;
+use Piwik\Config;
+use Piwik\Updater;
+use Piwik\Updates as PiwikUpdates;
+use Piwik\Updater\Migration\Factory as MigrationFactory;
+
+/**
+ * Update for version 5.0.0-b1.
+ */
+class Updates_5_0_0_b1 extends PiwikUpdates
+{
+    /**
+     * @var MigrationFactory
+     */
+    private $migration;
+
+    public function __construct(MigrationFactory $factory)
+    {
+        $this->migration = $factory;
+    }
+
+    public function getMigrations(Updater $updater)
+    {
+        $migrations = [];
+
+        // force utf8mb4 conversion in Matomo 5
+        if (DbHelper::getUsedCharset() !== 'utf8mb4') {
```

Comment:
@sgiehl is there a reason to force it in Matomo 5? Is it easy to still support utf8 basically or is there extra code for it?

Reply:
It's basically only the replacement of unsupported chars. We can also remove the update for 5.0. and create a new issue to reconsider that later...

Reply:
Be great to generally just perform the migration to utf8mb4 again just like in Matomo 4, but not do it if it isn't possible.